lucene/core/src/test/org/apache/lucene/index/TestDocValues.java

```diff
@@ -123,8 +124,72 @@ public void testNumericField() throws Exception {
     iw.close();
     dir.close();
   }
-  
-  /** 
+
+  // The LUCENE-8585 jump-tables enables O(1) skipping of IndexedDISI blocks, DENSE block lookup
+  // and numeric multi blocks. This test focuses on testing these jumps.
+  @Slow
+  public void testNumericFieldJumpTables() throws Exception {
+    Directory dir = newDirectory();
+    IndexWriter iw = new IndexWriter(dir, newIndexWriterConfig(null));
+    final int maxDoc = atLeast(3*65536); // Must be above 3*65536 to trigger IndexedDISI block skips
```

Comment:
Why do we have this test here as opposed to TestLucene80DocValuesFormat?

Given that you are indexing so many documents and don't care about testing indexing as much as you care about checking iterators, you should make sure that the index writer config isn't too crazy and enforce sane values of maxBufferedDocs, RAMBufferSizeMB, and a sane merge policy (eg. not the alcoholic one). See for instance TestLucene70DocValuesFormat#doTestSortedNumericBlocksOfVariousBitsPerValue.